apps/web/app/environments/[environmentId]/surveys/SurveyList.tsx

```diff
@@ -174,13 +174,13 @@ export default function SurveysList({ environmentId }) {
                     className="font-base"></Badge>
                   <p className="my-2 line-clamp-3 text-lg">{survey.name}</p>
                 </div>
-                <Link
```

Comment:
Why did you replace the Link with a?

Reply:
On repeatedly switching between surveys page and individual survey page, there was an issue of title not getting set because generateMetadata function in individual in layout.tsx does not run every time page is rendered and hence results into title mismatch, but using 'a' tag refreshes full page and hence sets appropriate title

Reply:
@mattinannt is this an issue?